Looks like a test for dependency inference failed because mlflow now depends on sklearn:

```
_____________________________ test_prune_packages ______________________________

    def test_prune_packages():
        assert _prune_packages(["mlflow"]) == {"mlflow"}
        assert _prune_packages(["mlflow", "packaging"]) == {"mlflow"}
>       assert _prune_packages(["mlflow", "scikit-learn"]) == {"mlflow", "scikit-learn"}
E       AssertionError: assert {'mlflow'} == {'mlflow', 'scikit-learn'}
E         Extra items in the right set:
E         'scikit-learn'
E         Full diff:
E         - {'scikit-learn', 'mlflow'}
E         + {'mlflow'}
```

This can be fixed easily.

--- a/requirements/core-requirements.txt
+++ b/requirements/core-requirements.txt
@@ -12,10 +12,10 @@ querystring_parser<2
 sqlalchemy<2,>=1.4.0
 gunicorn<21; platform_system != 'Windows'
 waitress<3; platform_system == 'Windows'
-scikit-learn<2
-pyarrow<10
-shap<1
-pandas-profiling<4
-ipython<9
-markdown<4
-Jinja2<4
+scikit-learn<2,>=1.0
+pyarrow<10,>=4.0.0
+shap<1,>=0.40
+pandas-profiling<4,>=3.1
+ipython<9,>=7.0
+markdown<4,>=3.3
+Jinja2<4,>=3.0

--- a/requirements/core-requirements.yaml
+++ b/requirements/core-requirements.yaml
@@ -49,28 +49,35 @@ waitress:
 
 scikit-learn:
   pip_release: scikit-learn
+  minimum: "1.0"
   max_major_version: 1
 
 pyarrow:
   pip_release: pyarrow
+  minimum: "4.0.0"
   max_major_version: 9
 
 shap:
   pip_release: shap
+  minimum: "0.40"
   max_major_version: 0
 
 pandas-profiling:
   pip_release: pandas-profiling
+  minimum: "3.1"
   max_major_version: 3
 
 ipython:
   pip_release: ipython
+  minimum: "7.0"
   max_major_version: 8
 
 markdown:
   pip_release: markdown
+  minimum: "3.3"
   max_major_version: 3
 
 Jinja2:
   pip_release: Jinja2
+  minimum: "3.0"
   max_major_version: 3

--- a/tests/utils/test_requirements_utils.py
+++ b/tests/utils/test_requirements_utils.py
@@ -200,7 +200,7 @@ def test_normalize_package_name():
 def test_prune_packages():
     assert _prune_packages(["mlflow"]) == {"mlflow"}
     assert _prune_packages(["mlflow", "packaging"]) == {"mlflow"}
-    assert _prune_packages(["mlflow", "scikit-learn"]) == {"mlflow", "scikit-learn"}
+    assert _prune_packages(["mlflow", "scikit-learn"]) == {"mlflow"}
 
 
 def test_capture_imported_modules():

